user_docs/en/userGuide.t2t

```diff
@@ -355,8 +355,8 @@ Portable and temporary copies of NVDA have the following restrictions:
 - The inability to automatically start during and/or after log-on.
 - The inability to interact with applications running with administrative privileges, unless of course NVDA itself has been run also with these privileges (not recommended).
 - The inability to read User Account Control (UAC) screens when trying to start an application with administrative privileges.
-- Windows 8 and later: the inability to support input from a touchscreen.
-- Windows 8 and later: the inability to provide features such as browse mode and speaking of typed characters in Windows Store apps.
+- The inability to support input from a touchscreen.
+- The inability to provide features such as browse mode and speaking of typed characters in Windows Store apps.
 - Windows 8 and later: audio ducking is not supported.

```

Comment:
Could this also be amended? As audioDucking has no explicit windows checks I think it's unlikely to be caught later
```suggestion
- Audio ducking is not supported.
```